saga/src/activities.ts

```diff
@@ -0,0 +1,17 @@
+import { BoundedContextClients } from './types/clients';
+
+// createActivities just returns client implementations directly right now, but later might decorate
+export function createActivities(clients: BoundedContextClients) {
+  return {
+    createAccount: clients.accounts.createAccount,
```

Comment:
Shouldn't these methods be bound to the instance?

Reply:
i mean, they can be, but in this case it really is just illustrating a common pattern of delegating to something like a HTTP client as an "activity" and compose all the clients funcs for use in workflows. you want me to change it to something in particular? Happy to do so

Reply:
The issue with the way you have it set up is that if someone extends this sample and tries to reference `this` in any of the client methods, they'll get the global object instead of the instance because the methods aren't bound.

You can bind them with:
```ts
createAccount: clients.accounts.createAccount.bind(clients.accounts)
-- OR --
createAccount: (...args: Parameters<typeof clients.accounts.createAccount>) => clients.accounts.createAccount(...args)
```

The former is obviously cleaner :)

Reply:
implemented